rootfs/usr/local/bin/init-terraform

```diff
@@ -8,6 +8,9 @@ TF_FILE=${TF_FILE:-terraform.tfstate}
 TF_FILES=(*.tf)
 TF_ARGS=("-backend=true" "-backend-config=key=${TF_BUCKET_PREFIX}/${TF_FILE}")
 
+# Set default plugin cache dir
+export TF_PLUGIN_CACHE_DIR="${TF_PLUGIN_CACHE_DIR:-/localhost/.terraform.d/plugins}"
```

Comment:
I don't think this is sufficient. This will work when you call `init-terraform` (which calls `terraform init`), but when you call `terraform plan` or `terraform apply`, will it know where to find them?

Reply:
Would this be a good spot? https://github.com/cloudposse/geodesic/blob/master/rootfs/etc/profile.d/terraform.sh

Reply:
hmm, good point. `/etc/profile.d/terraform.sh` isn't going to work when the Atlantis user has `/bin/false` shell